app/dev-portal/portals/customization/snippets.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Snippets
+---
+
+Snippets enable reuse of content and MDC components across [Pages](/dev-portal/portals/customization/custom-pages). Visibility controls and Publishing status allow you to enable/disable Snippets, and/or restrict access to logged-in Developers.
+
+To get started creating Snippets, navigate to your Dev Portal and select [**Portal Editor**](/dev-portal/portals/customization/portal-editor/), then Snippets from the left sidebar.
+
+{:.note}
+> *Snippets are built using Markdown Components (MDC). Additional documentation on Snippet syntax and usage is available on a [dedicated MDC site](https://portaldocs.konghq.com/components/snippet).*
+
+
+### Create a new Snippet
+* Click `+` at the top
```

Comment:
suggestion: similar to what Travis mentioned, can we just reference "Click to create a new Snippet" instead of specific instructions in case the UX changes over time?